### PR TITLE
MSA-3022: Add ThirdReality zigbee switch support.

### PR DESCRIPTION
**Pull request by XQ-Huang**
Submitting 'Add ThirdReality zigbee switch support.' for publication

### CODE REVIEW

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 1]
```diff
@@ -13,7 +13,7 @@
  */
 
 metadata {
-    definition (name: "ZigBee Switch", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch", runLocally: true, minHubCoreVersion: '000.019.00012', executeCommandsLocally: true) {
+    definition (name: "ZigBee Switch", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.switch") {
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: For your testing you can comment out or remove this code but please leave it as is for the code review

File: devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy [hunk 2]
```diff
@@ -25,9 +25,8 @@ metadata {
...
         fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0003, 0006, 0008, 0019, 0406", manufacturer: "Leviton", model: "DL15S", deviceJoinName: "Leviton Lumina RF Switch"
-        fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, 1000, 0B04, FC0F", outClusters: "0019", manufacturer: "OSRAM", model: "Plug 01", deviceJoinName: "OSRAM SMART+ Plug"
-        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0B05, FC01, FC08", outClusters: "0003, 0019", manufacturer: "LEDVANCE", model: "PLUG", deviceJoinName: "SYLVANIA SMART+ Smart Plug"
-        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0B05", outClusters: "0019", manufacturer: "sengled", model: "E1C-NB6", deviceJoinName: "Sengled Element Outlet"
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0019", manufacturer: "Third Reality, Inc", model: "3RSS008Z", deviceJoinName: "RealitySwitch2"
```
Comment (tpmanley) [inline devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Please add the new fingerprints to the end of the list and leave the existing fingerprints

Comment (tpmanley) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: Is RealitySwitch2 the name you want displayed when this device joins?

Comment (XQ-Huang) [reply devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy]: There are two devices need to be commented.
The one model: "3RSS007Z" should displayed as "RealitySwitch" and another model: "3RSS008Z" should displayed as "RealitySwitch Plus".

**Review by tpmanley — APPROVED** [review]

**Pull request merged**